scene/gui/text_edit.cpp

```diff
@@ -428,10 +428,10 @@ void TextEdit::_notification(int p_what) {
 		case NOTIFICATION_ENTER_TREE: {
 			_update_caches();
 			if (caret_pos_dirty) {
-				MessageQueue::get_singleton()->push_call(this, "_emit_caret_changed");
+				callable_mp(this, &TextEdit::_emit_caret_changed).call_deferred();
```

Comment:
Since this used this syntax should it be `MessageQueue::get_singleton()->push_callable(callable_mp(this, &TextEdit::_emit_caret_changed));`?

Reply:
No, I changed MessageQueue calls to Callables to.

Reply:
Unit tests fail for this class worth checking if using the message queue for these solves that

Reply:
If needed, MessageQueue can still be used with callables, `MessageQueue::get_singleton()->push_callable` is a thing :)

Reply:
Did not resolve this, unsure what got broken here

Reply:
I need to double-check all removed binds. `callable_mp()` does not support default arguments 😓